pages/new-route.js

```diff
@@ -1,6 +1,22 @@
+import Link from "next/link";
+
 const baseJourneyURL = "https://api.tfl.gov.uk/Journey/JourneyResults/";
+import Cookies from "cookies";
+
+import { getSession } from "../database/model";
 
 export async function getServerSideProps(params) {
+  const cookieSigningKeys = [process.env.COOKIE_SECRET];
+
+  const cookies = new Cookies(params.req, params.res, {
+    keys: cookieSigningKeys,
+  });
+
+  const sidCookie =
+    cookies.get("sid", { signed: true, sameSite: "strict" }) || null;

```

Comment:
I can't remember cookies much but does signed mean we don't need httpOnly?

Reply:
In the session auth workshop https://learn.foundersandcoders.com/workshops/session-auth/ COOKIE_OPTIONS uses both signed and httpOnly